codegen/xc_fallback_generate/generate.jl

```diff
@@ -0,0 +1,48 @@
+# Adapted from the maple scripts in libxc
```

Comment:
Could you add a README to that dir explaining the rationale here? I don't really get what this generates, and why this is needed. Also what's vwn?

Reply:
Sure. Will do if you think it's worth it.

The point is that libxc actually uses an analytic procedure (written in Maple) to generate the C code for computing exchange correlation functionals. I.e. you only code the energy expression in a general form and the special cases and the potential / higher-order derivatives are autogenerated. So instead of recoding the LDA expressions in Julia from the papers, I wanted to see how difficult it is to use the Maple scripts and generate the expressions automatically in a useful way. It's not completely clean (in the sense that one has to manually adapt afterwards) and probably at present will not scale to the complete repository of functionals available in libxc, but if we need some selected functionals in different precision than `Float64`, this generator is probably useful.

VWN = Vosko Wilk,and Nusair. That's the term commonly used in quantum chemistry literature to refer to [this paper](https://doi.org/10.1139/p80-159), where they compare different parametrisations of LDA exchange. The "standard one" to the best of my knowledge is the one with a Pade approximation of a numerical diffusion Monte-Carlo simulation and that's the one I use as well.

Reply:
Just put that in the README :p

Reply:
Done.